near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -272,10 +274,33 @@ impl NonFungibleToken {
         self.internal_transfer_unguarded(token_id, &owner_id, receiver_id);
 
         log!("Transfer {} from {} to {}", token_id, sender_id, receiver_id);
-        if let Some(memo) = memo {
+        if let Some(memo) = memo.as_ref() {
             log!("Memo: {}", memo);
         }
 
+        //default the authorized ID to be None for the logs
+        let mut authorized_id = None; 
+        //if the approval ID was provided, set the authorized ID equal to the sender
+        if !approval_id.is_none() {
+            authorized_id = Some(sender_id.to_string()); 
+        }
+
+        //construct the mint log as per the events standard
```

Comment:
Clippy is complaining about style such as the following:
```suggestion
        // construct the mint log as per the events standard
```
Recommended you run `cargo fix` and `cargo format`, then check afterwards to see if it fixed everything with `cargo clippy`

Reply:
will do - didn't know that was a thing sorry :)

Reply:
done